ImageManager/ImageManager.py

```diff
@@ -0,0 +1,89 @@
+"""
+    Handles CRUD operations for images, and their configuration on the
+    FIWARE backend
+"""
+
+import json
+import logging
+from time import time
+from flask import request
+from flask import make_response
+from flask import Blueprint
+from utils import *
+
+from DatabaseModels import *
+from SerializationModels import *
+from TenancyManager import init_tenant_context
+
+from app import app
+
+image = Blueprint('image', __name__)
+
+LOGGER = logging.getLogger('image-manager.' + __name__)
+LOGGER.addHandler(logging.StreamHandler())
+LOGGER.setLevel(logging.INFO)
+
+global_imageid = 20
+
+
+@image.route('/image/<imageid>', methods=['GET'])
+def get_image(imageid):
+    try:
+        print(request)
```

Comment:
I think it'd be better to make use of LOGGER for log generation.
If this is just left over from development, we could remove it.